### PR TITLE
Don't display buttons if there are no system notifications

### PR DESCRIPTION
**Pull request by adelowo**
See attached images
screen shot 2018-06-19 at 23 14 44
screen shot 2018-06-19 at 23 15 26

### CODE REVIEW

File: templates/admin/notice.tmpl [hunk 1]
```diff
@@ -97,4 +99,4 @@
 		<p></p>
 	</div>
 </div>
-{{template "base/footer" .}}
+{
```
Comment (lafriks) [inline templates/admin/notice.tmpl]: this should be changed back to `{{template "base/footer" .}}`

**Review by lafriks — CHANGES_REQUESTED** [review]

File: templates/admin/notice.tmpl (hunk 1 above)
Comment (adelowo) [reply templates/admin/notice.tmpl]: Done @lafriks

File: templates/admin/notice.tmpl [hunk 2]
```diff
@@ -35,31 +35,33 @@
...
-								<div class="menu">
-									<div class="item select action" data-action="select-all">
-										{{.i18n.Tr "admin.notices.select_all"}}
-									</div>
-									<div class="item select action" data-action="deselect-all">
-										{{.i18n.Tr "admin.notices.deselect_all"}}
-									</div>
-									<div class="item select action" data-action="inverse">
-										{{.i18n.Tr "admin.notices.inverse_selection"}}
+					{{ if gt (len .Notices) 0 }}
```
Comment (lafriks) [inline templates/admin/notice.tmpl]: Also I think this `{{ if ... }}` should be before `<tfoot class="full-width">` and `{{ end }}` after `</tfoot>`

Comment (adelowo) [reply templates/admin/notice.tmpl]: Added that already and rebased

**Review by lafriks — APPROVED** [review]

**Comment by lunny** [comment]
Does github have the same behavior?

**Comment by lafriks** [comment]
@lunny github does not have such buttons at all, so I think it is ok to hide them when they are not needed anyway

File: templates/admin/notice.tmpl [hunk 3]
```diff
@@ -34,33 +34,35 @@
...
-						<th colspan="5">
-							<div class="ui right">
-								<a class="ui red small button" href="{{AppSubUrl}}/admin/notices/empty">{{.i18n.Tr "admin.notices.delete_all"}}</a>
-							</div>
-							<div class="ui floating upward dropdown small button">
-								<span class="text">{{.i18n.Tr "admin.notices.actions"}}</span>
-								<div class="menu">
-									<div class="item select action" data-action="select-all">
-										{{.i18n.Tr "admin.notices.select_all"}}
+				{{ if gt (len .Notices) 0 }}
```
Comment (jonasfranz) [inline templates/admin/notice.tmpl]: Please use `{{ if .Notices }}` instead

**Review by jonasfranz — CHANGES_REQUESTED** [review]

**Review by jonasfranz — APPROVED** [review]

**Pull request merged**